File: Python/Mahasiswa.py
class Mahasiswa:

    #public
    nama = ""
    prodi = ""
    fakultas = ""
    nim = ""

    #KONSTRUKSI
    def __init__(self):
        self.nama = ""
        self.prodi = ""
        self.fakultas = ""
        self.nim = ""

    #GETTER & SETTER
    #SETTER
    def setNama(self, nama):
        self.nama = nama  
    
    def setProdi(self, prodi):
        self.prodi

    def setFakultas(self, fakultas):
        self.fakultas

    def setNim(self, nim):
        self.nim
    #GETTER
    def getNama(self):
        return self.nama
    
    def getProdi(self):
        return self.prodi
    
    def getFakultas(self):
        return self.fakultas
    
    def getNim(self):
        return self.nim
    

File: C++/Mahasiswa.cpp
#include<iostream>
#include<string>

using namespace std;

class Mahasiswa{
    private:
        string nama, prodi, fakultas;
        int nim;

    public:
        //konstruksi
        Mahasiswa(){

        }

        /*GETTER and SETTER*/
        void setNama(string nama){
            this->nama = nama;
        }

        void setProdi(string prodi){
            this->prodi = prodi;
        }

        void setFakultas(string fakultas){
            this->fakultas = fakultas;
        }

        void setNim(int nim){
            this->nim = nim;
        }

        string getNama(){
            return this->nama;
        }

        string getProdi(){
            return this->prodi;
        }

        string getFakultas(){
            return this->fakultas;
        }

        int getNim(){
            return this->nim;
        }

        ~Mahasiswa(){

        }
};